interfaces/Portalicious/src/app/services/http-wrapper.service.ts

```diff
@@ -66,12 +78,60 @@ export class HttpWrapperService {
         const expires = Date.parse(user.expires);
 
         if (expires < Date.now()) {
-          return of(new Error('Token expired'));
+          return of(new Error($localize`:@@error-token-expired:Token expired`));
         }
       }
 
-      return of(new Error('Not authorized'));
+      return of(new Error($localize`:@@error-not-authorized:Not authorized`));
     }
+
+    let errorMessage = get(error, 'message', undefined);
```

Comment:
I added a lot of stuff here for default error handling. I'm not sure it's relevant always but I guess we'll find out through time?